### PR TITLE
Events clean up

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

File: packages/synthetix-main/contracts/interfaces/ILiquidationModule.sol [hunk 1]
```diff
@@ -3,13 +3,19 @@ pragma solidity ^0.8.0;
...
+
     event Liquidation(
         uint128 indexed accountId,
         uint128 indexed poolId,
         address indexed collateralType,
-        uint debtLiquidated,
-        uint collateralLiquidated,
-        uint amountRewarded
+        LiquidationData liquidationData,
+        uint liquidateAsAccountId,
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/interfaces/ILiquidationModule.sol:17]: supposed to be uint128

File: packages/synthetix-main/contracts/interfaces/ILiquidationModule.sol [hunk 2]
```diff
@@ -3,13 +3,19 @@ pragma solidity ^0.8.0;
...
+        address sender
     );
 
     event VaultLiquidation(
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/interfaces/ILiquidationModule.sol:21]: this structure should be updated to use the same `LiquidationData` structure as well

File: packages/synthetix-main/contracts/interfaces/IMarketManagerModule.sol [hunk 3]
```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 
 /// @title Market Manager Module
 interface IMarketManagerModule {
-    event MarketRegistered(address indexed market, uint128 indexed marketId);
+    event MarketRegistered(address indexed market, uint128 indexed marketId, address indexed sender);
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/interfaces/IMarketManagerModule.sol:6]: feels like sender might make more sense as the first property here

File: packages/synthetix-main/contracts/interfaces/IPoolModule.sol [hunk 4]
```diff
@@ -6,9 +6,9 @@ import "../storage/MarketConfiguration.sol";
 /// @title Module for managing pool token and pools positions distribution
 interface IPoolModule {
     /// @notice gets fired when pool will be created
-    event PoolCreated(uint128 indexed poolId, address indexed owner);
+    event PoolCreated(uint128 indexed poolId, address indexed owner, address indexed sender);
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/interfaces/IPoolModule.sol:9]: same thing, sender probably makes more sense in the front here

**Review by dbeal-eth — CHANGES_REQUESTED** [review]

File: packages/synthetix-main/contracts/interfaces/IMarketManagerModule.sol (hunk 3 above)
Comment (noahlitvin) [reply packages/synthetix-main/contracts/interfaces/IMarketManagerModule.sol:6]: I think we should keep this consistent with the other events.

**Review by leomassazza — APPROVED** [review]

**Review by dbeal-eth — APPROVED** [review]

**Pull request merged**